<?xml version="1.0"?>
<launch>

    <node name="perception" pkg="perception_pkg" type="perception"  output="screen">
        <!-- E区任务 -->
        <param name="D_x_max" value="15"/>
        <param name="D_x_min" value="11"/>
        <param name="D_y_max" value="2.0"/>
        <param name="D_y_min" value="-2.0"/>
        <param name="D_z_max" value="1.5"/>
        <param name="D_z_min" value="0.1"/>
        <param name="dist_threshold" value="0.3"/>
        <param name="max_iteration" value="5000"/>
        <param name="radius_real" value="0.6"/>
        <param name="acceptable_error" value="0.3"/>     

        <!-- B区任务 -->
        <param name="B_x_max" value="14.0"/>
        <param name="B_x_min" value="10.0"/>
        <param name="B_y_max" value="1.0"/>
        <param name="B_y_min" value="-5.0"/>
        <param name="B_z_max" value="2.7"/>
        <param name="B_z_min" value="0.5"/>
        <param name="search_radius" value="0.7"/>
        <param name="min_neighbors_num" value="5"/>
        <param name="max_width" value="1.75"/>
        <param name="min_width" value="1.25"/>
        
    </node>

</launch>
